src/main/app.js

```diff
@@ -78,18 +78,15 @@ if (isWindows()) {
 }
 
 // Someone tried to run a second instance, we should focus our window.
-const isSecondInstance = app.makeSingleInstance(() => {
+app.requestSingleInstanceLock();
+app.on('second-instance', (event, argv, cwd) => {
```

Comment:
Quick question, no request here: So this means that a new instance won't open? That's good, but I guess if we want to notify the user we need to update this later so that a newer version could run?

Reply:
@perry-mitchell I think so, I think I'll do that in the new version. Requires some more coding